vendor/brave_base/BUILD.gn

```diff
@@ -0,0 +1,26 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+config("external_config") {
+  include_dirs = [ "../.." ]
+}
+
+config("internal_config") {
```

Comment:
there's really no need to have an internal config here. We only need that when we have public/internal code we want to separate

Reply:
done